astropy/constants/constant.py

```diff
@@ -159,10 +159,10 @@ def name(self):
         return self._name
 
     @lazyproperty
-    def unit(self):
+    def _unit(self):
```

Comment:
I'm not entirely sure I follow though.  Why do we need the property at all if this is also handled by `Quantity` now?